test/Microsoft.VisualStudio.Threading.Analyzers.Tests/Helpers/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -79,7 +79,7 @@ where resourceName.StartsWith(additionalFilePrefix, StringComparison.Ordinal)
 
             protected override ParseOptions CreateParseOptions()
             {
-                return ((CSharpParseOptions)base.CreateParseOptions()).WithLanguageVersion(LanguageVersion.CSharp8);
+                return ((CSharpParseOptions)base.CreateParseOptions()).WithLanguageVersion(LanguageVersion.CSharp10);
```

Comment:
I bumped the language version in order to use an explicit return type in the async void lambda unit test's test code.